ckanext/privatedatasets/tests/test_actions.py

```diff
@@ -92,6 +93,17 @@ def test_class_cannot_be_loaded(self, class_path, class_name, path_exist, class_
         # Checks
         self.assertEquals(0, actions.plugins.toolkit.get_action.call_count)
 
+        if expected_error:
+            with self.assertRaises(actions.plugins.toolkit.ValidationError) as cm:
+                actions.package_deleted({},{})
+            self.assertEqual(cm.exception.error_dict['message'], expected_error)
+        else:
+            # Exception is not risen
+            self.assertEquals(None, actions.package_deleted({},{}))
+
+        # Checks
+        self.assertEquals(0, actions.plugins.toolkit.get_action.call_count)
```

Comment:
You are doing something strange here. You are testing that the methods raise an exception when something cannot be imported. 

In the line 87, you have included the actions.package_deleted that is not going to never being called since the previous one raises and exception.

Then you have copied and pasted the raise assertion for the package_deleted. You should refactor this code, and test the different methods in different tests (of course don't copy and paste this one, reuse!)